unstructured/ingest/connector/airtable.py

```diff
@@ -0,0 +1,180 @@
+import os
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Optional
+
+from pyairtable import Api
+from pyairtable.metadata import get_api_bases, get_base_schema
```

Comment:
these should probably only be imported inline where needed (with the function wrapped with the @requires_dependencies(["pyairtable"])

Reply:
Addressed! Noted down to refactor it in other connectors as well 👍